.github/workflows/ci.yml

```diff
@@ -45,15 +45,6 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run YAML lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-YAML-Lint@v1
-  copypaste-detector:
```

Comment:
should we remove this action?

Reply:
Oh, this seems to have been a rebase problem. I'll take care of it. Thanks for noticing this!

Reply:
Fixed and repushed.